docker/configure_workers_and_start.py

```diff
@@ -163,6 +164,7 @@
             "^/_matrix/federation/(v1|v2)/invite/",
             "^/_matrix/federation/(v1|v2)/query_auth/",
             "^/_matrix/federation/(v1|v2)/event_auth/",
+            "^/_matrix/federation/(v1|v2)/timestamp_to_event",
```

Comment:
I assume we might care about these more now that `/timestmap_to_event` is stable.

Perhaps just an oversight that these weren't here before?

Reply:
Does this really have both a v1 and v2 version or is that a copy and paste error?

```suggestion
            "^/_matrix/federation/v1/timestamp_to_event",
```

Please update https://matrix-org.github.io/synapse/develop/workers.html to match what endpoints can be used on workers.

Reply:
Does this look right? https://github.com/matrix-org/synapse/pull/14471/commits/e702a5d15c709f0fb38611ceaa45242afa243cdd

Reply:
Yes, looks good. Thanks!